save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/CreateVulnerabilityView.kt

```diff
@@ -142,6 +173,47 @@ val createVulnerabilityView: VFC = VFC {
                                     }
                                 }
 
+                                inputTextFormOptional {
+                                    form = InputTypes.GIT_URL
+                                    textValue = vulnerability.relatedLink
+                                    validInput = conflictErrorMessage.isNullOrEmpty()
+                                    classes = "col-md-12 mt-3 mb-3 pl-2 pr-2 text-left"
+                                    name = "Related link"
+                                    onChangeFun = { event ->
+                                        setVulnerability { it.copy(relatedLink = event.target.value) }
+                                    }
+                                }
+
+                                div {
+                                    className = ClassName("col-md-12 pl-2 pr-2 mt-3 text-left align-self-center")
+                                    +"Language"
+
+                                    div {
+                                        className = ClassName("input-group")
+                                        ReactHTML.select {
```

Comment:
again and again :)

Why do you use `select` btw? We have `selectBuilder` or smth like that